src/main/java/org/candlepin/subscriptions/ApplicationProperties.java

```diff
@@ -165,4 +165,12 @@
      */
     private Duration metricLookupRangeDuration = Duration.ofHours(24L);
 
+    /**
+     * Latency offset: how many minutes in the past you would like the metering job latency (window) to occur.
+     *
+     * The offset is subtracted from the beginning and ending times of the latency window, to delay the entire
+     * processing window.  This ensures more metering tasks finish and report their totals before tallying
+     * begins.
+     */
+    private Duration hourlyTallyOffsetMinutes = Duration.ofMinutes(60L);
```

Comment:
Please refactor the name:

s/hourlyTallyOffsetMinutes/hourlyTallyOffset/

Until I did this locally, the env var did not work as expected.

Reply:
Ah, crud.  My fault.  I will change this.